### PR TITLE
Fix line folding

### PR DESCRIPTION
**Pull request by artursmirnov**
Trying to fix https://github.com/travis-ci/travis-web/pull/2102

### CODE REVIEW

File: app/styles/app/layouts/log.scss [hunk 1]
```diff
@@ -246,13 +246,13 @@
       position: absolute;
       padding-right: 1em;
     }
```
Comment (piotr-travisci) [inline app/styles/app/layouts/log.scss]:
```suggestion
      -webkit-user-select: none;
      -moz-user-select: none;
      -ms-user-select: none;
      user-select: none;
    }
```

Comment (artursmirnov) [reply app/styles/app/layouts/log.scss]: @piotr-travisci this is a great suggestion! Though, prefixes are added automatically by [autoprefixer](https://github.com/travis-ci/travis-web/blob/master/package.json#L34) and thus not necessary.

Comment (piotr-travisci) [reply app/styles/app/layouts/log.scss]: 👍 @artursmirnov.
Suggestion refactored.

Comment (artursmirnov) [reply app/styles/app/layouts/log.scss]: Thanks!

**Pull request merged**